src/controller/clientUsage.ts

```diff
@@ -14,6 +14,17 @@ export class ClientUsageController {
 
 	async registerUsage(buffer: Buffer): Promise<void> {
 		const { tracesPerQuery } = Report.decode(buffer).toJSON() as IReport;
+
+		if (!tracesPerQuery) {
+			return;
+		}
+		const isGraphQlFailure = Object.keys(tracesPerQuery).filter((trace) =>
+			trace.startsWith('## GraphQLValidationFailure')
```

Comment:
Could you please add `GraphQLParseFailure`  as well? 
https://github.com/apollographql/router/blob/4b6f62a41bf539c63bdc1dfe17c79c20203e40c6/apollo-router/src/spec/mod.rs#L44

Reply:
Done